From 81a3072d6a0b06f8962660d8ea367129e50aa8f8 Mon Sep 17 00:00:00 2001
From: Isaku Yamahata <isaku.yamahata@intel.com>
Date: Thu, 26 May 2022 17:12:20 -0700
Subject: [PATCH 68/99] [REVERTME] bump KVM_EXIT_MEMORY_FAULT to avoid merge
 conflict

To avoid merge conflict with the upstream, bump KVM_EXIT_MEMORY_FAULT
instead of using the next unused value.
TODO: once the patch is upstream, remove this patch and correct
KVM_EXIT_MEMORY_FAULT.

Signed-off-by: Isaku Yamahata <isaku.yamahata@intel.com>
---
 linux-headers/linux/kvm.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linux-headers/linux/kvm.h b/linux-headers/linux/kvm.h
index b48723d460..62c722c9f0 100644
--- a/linux-headers/linux/kvm.h
+++ b/linux-headers/linux/kvm.h
@@ -334,9 +334,9 @@ struct kvm_tdx_exit {
 #define KVM_EXIT_X86_BUS_LOCK     33
 #define KVM_EXIT_XEN              34
 #define KVM_EXIT_RISCV_SBI        35
-#define KVM_EXIT_MEMORY_FAULT     36
 /* WORKAROUND to avoid conflict with upstream. */
 #define KVM_EXIT_TDX              50
+#define KVM_EXIT_MEMORY_FAULT     100
 
 /* For KVM_EXIT_INTERNAL_ERROR */
 /* Emulate instruction failed. */
-- 
2.31.1

